### DISCUSSION TITLE
Why is fetching partial entities supported only for SELECT_IN loading strategy?

### DISCUSSION THREAD

**Discussion by hayksaryan** [mikro-orm/mikro-orm#4398] [Q&A]
Are there any technical limitations/blockers, or is it planned to be implemented in future releases?

**Comment by B4nan** [comment]
Why do you say so? It is supported on joined strategy for quite some time (months, if not the whole v5 lifecycle).

https://github.com/mikro-orm/mikro-orm/blob/master/tests/features/partial-loading/partial-loading.mysql.test.ts#L202

**Reply by hayksaryan** [reply]
Yes, I though I was already using it for quite some time, it's just that docs still have a note saying that it's only supported for SELECT_IN strategy. Will probably need to update then.
[Docs link](https://mikro-orm.io/docs/entity-manager#fetching-partial-entities)
![Doc screenshot](https://github.com/mikro-orm/mikro-orm/assets/52959001/c7e175b1-9c7e-43a7-99ee-245ecec3b158)
